client/galaxy/scripts/mvc/tool/tool-genomespace.js

```diff
@@ -0,0 +1,23 @@
+// Provides support for interacting with the GenomeSpace File Browser popup dialogue
+define([], function() {
+
+// tool form templates
+return {
+    openFileBrowser: function( options ) {
+        var GS_UPLOAD_URL = "https://gsui.genomespace.org/jsui/upload/loadUrlToGenomespace.html?getLocation=true";
```

Comment:
Galaxy.config may be available - that is like a global configuration object that you can populate with options from galaxy.ini and such. It is worth looking into.